ViewRenderer.php

```diff
@@ -248,6 +248,8 @@ private function _addCustom($classType, $elements)
 
             if ($twigElement !== null) {
                 $this->twig->{'add'.$classType}($name, $twigElement);
+            } elseif (is_object($func) && in_array(get_class($func), ['Twig_SimpleFunction', 'Twig_SimpleFilter'])) {
```

Comment:
Could be replaced with `$func insanceof \Twig_SimpleFunction || $func instanceof \Twig_SimpleFilter`.